web/src/components/Form/Expenses/ExpenseCategory.js

```diff
@@ -0,0 +1,101 @@
+// import PropTypes from 'prop-types';
+import React from 'react';
+import { Box, Button } from 'rebass';
+import { Field, FieldArray, reduxForm } from 'redux-form';
+
+import { Input } from '../../Inputs';
+import SectionHeader from '../../SectionHeader';
+
+const ExpenseCategory = () => (
+  <Box py={4}>
```

Comment:
for accessibility, should these be wrapped in a `form` element?

Reply:
No idea.  Probably?  😬

Reply:
i'm not sure it's a accessibility requirement, but it's good html semantics